src/loaders/gtdb/gtdb_taxa_freq_loader.py

```diff
@@ -91,8 +91,8 @@ def main():
                         help='GTDB taxonomy files')
 
     # Required flag argument
-    parser.add_argument('--load_version', required=True, type=str, nargs=1,
-                        help='GTDB load version')  # TODO parse from load_files name
```

Comment:
I don't think we actually want to do this TODO since the load versions could be things like
```
207-dev
207.kbase.1
207.kbase.2
```